test/gtest/ucp/test_ucp_wireup.cc

```diff
@@ -485,7 +485,9 @@ UCS_TEST_P(test_ucp_wireup_1sided, stress_connect) {
 }
 
 UCS_TEST_P(test_ucp_wireup_1sided, stress_connect2) {
-    int count = ucs_min(1000 / ucs::test_time_multiplier(), max_connections() / 2);
+    int count = ucs_min(1000 / ucs::test_time_multiplier() *
```

Comment:
can this be simplified somehow? Looks overcomplicated.

Reply:
also, 1000 / ucs::test_time_multiplier() * ucs::test_time_multiplier() is just 1000 ...